tools/esdlcmd/README.md

```diff
@@ -0,0 +1,485 @@
+
+Second are binding or service-specific attributes. This includes any attribute not in the other categories. These are all output on the `Binding/Definition` element. This category will likely grow over time but currently is just `@auth_feature`:
+
+| Attribute   | Required? | Value  | Usage |
+| - | :-: | - | - |
+| `@auth_feature` | No | string | Used declare authorization settings if they aren't present in the ESDL Definition, or override them if they are. Additional documentation on this attribute is forthcoming. |
+
+Finally are auxillary attributes. These should be thought of as read-only or for reference, and it is not recommended that you set these in the manifest. They are set by the system when publishing to dali using `esdl bind-service`, and they are present on binding configurations retrieved from the dali. If you set these attributes in the manifest, the values will be overwritten when running `esdl bind-service`. Alternately, if you run as an esdl application, these values aren't set by default and don't have a material effect on the binding, but they may appear in the trace log.
+
+| Attribute   | Required? | Value  | Usage |
+| - | :-: | - | - |
+| `@created`     | No | string | Timestamp of binding creation |
+| `@espbinding`  | No | string | Set to match the `id`. Otherwise the value is unset and unused. |
+| `@espprocess`  | No | string | Name of the ESP process this binding is running on |
+| `@id`          | No | string | Runtime name of the binding. When publishing to dali the value is [ESP Process].[port].[ESDL Service]. When not present in the manifest a default value is generated of the form [@esdlservice]_desdl_binding|
+| `@port`        | No | string | Port on the ESP Process listening for connections to the binding. |
+| `@publishedBy` | No | string | Userid of the person publishing the binding |
+
+The `<em:ServiceBinding>` element is not required since it is possible to embed a complete `Binding` element hierarchy in the manifest.
+
+#### EsdlDefinition
+Recommended child of `<em:Manifest>` that enables ESDL definition-specific logic in the tool:
+
+- A `Definitions` element is output. All content is enclosed in a CDATA section.
+- The manifest `<em:Include>` element is recognized to import ESDL definitions:
+  - An included .ecm file is transformed into XML.
+  - An included .xml file is imported as-is.
+
+| Attribute   | Required? | Value  | Usage |
+| - | :-: | - | - |
+| N/A |
+
+The element is not required since it is possible to embed a complete `Definitions` element hierarchy in the manifest.
+
+#### Include
+Optional element that imports the contents of another file into the output, in place of the itself. The outcome of the import depends on the context in which this element is used. See [EsdlDefinition](#esdldefinition), [Scripts](@scripts), and [Transform](#transform) for more information.
```

Comment:
"... in place of the itself." Remove "the"

Reply:
fixed